### DISCUSSION TITLE
How to configure Tasmota ZB Bridge (MQTT) connected Dimmer switch in Home Assistant

### DISCUSSION THREAD

**Discussion by rebel7580** [arendst/Tasmota#11548] [Q&A] [answered]
I have a Sonoff ZB Bridge flashed and working with Tasmota 9.3.1.1, running MQTT mode.

I have a Enbrighten Dimmer switch (Jasco #43080) connected on the Zigbee network.
I can access and control this dimmer via my (legacy) home automation system, but I also run Home Assistant. I'd like to get this Dimmer configured in HA, but I'm having problems figuring out the correct way to handle it. The stumbling block for me is the fact that the dimmer:

1) reports power and brightness status changes (sometimes) separately using ZbReceived
2) Sets on/off and brightness values using separate ZbSend commands in a different format than in #1.

I've tried basic configurations, those using template and json schema, etc., and have not yet come up with a solution that HA is happy with.

Has anyone gotten an MQTT-based configuration.yaml working for these Zigbee dimmers?

BTW, I have some Enbrighten non-dimming switches (Jasco #43076) as well and I use the following to successfully configure them in yaml:
<pre>
  - platform: mqtt
    unique_id: "ZB_UnderCabinet"
    name: "Under Cabinet"
    command_topic: "cmnd/tasmota_zbBridge/6D52/ZbSend"
    payload_on: '{"device":"0x6D52","send":{"POWER":"On"}}'
    payload_off: '{"device":"0x6D52","send":{"POWER":"Off"}}'
    state_topic: "tele/tasmota_zbBridge/6D52/ZbReceived"
    value_template: "{{ value_json.Power }}"
    state_on: 1
    state_off: 0
    qos: 1
</pre>

**Comment by ascillato2** [comment, marked as answer]
Please, address this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4). The chat is a better and more dynamic channel for helping you.

Thanks.

---------------------------

### Support Information 
See [Docs](https://tasmota.github.io/docs/) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
See [Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)

**Reply by rebel7580** [reply]
I filed this question here (in Q&A!) because I believed this would be a more complex discussion than is good for discord.  If not, what exactly is this Q&A discussion forum for??  Then again, no one responded here either. :(

**Reply by ascillato** [reply]
I was just giving you another extra channel, due to your question didn't receive any reply here.

Sorry that you didn't receive any hint there too. Please, keep trying until you find someone that can help you.

**Reply by rebel7580** [reply]
Fair enough. Thanks!
